templates/bundles/DemosPlanStatementBundle/DemosPlanStatement/includes/fragment_statement_form.html.twig

```diff
@@ -66,7 +66,7 @@
                 >
                     <template v-slot:option="props">
                         <span v-if="props.option.$isLabel">
-                            {% verbatim %}{{ props.option.$groupLabel }}{% endverbatim %}
+                            <strong>{% verbatim %}{{ props.option.$groupLabel }}{% endverbatim %}</strong>
```

Comment:
do we need the span anymore, if we have the strong-tag?

Reply:
no we don't. 80d6486b10866d507b002257318005e659a45790